Dockerfile

```diff
@@ -133,7 +134,7 @@ COPY --from=generate-build /api/cluster/*.pb.go /pkg/machinery/api/cluster/
 COPY --from=generate-build /api/storage/*.pb.go /pkg/machinery/api/storage/
 COPY --from=generate-build /api/resource/*.pb.go /pkg/machinery/api/resource/
 COPY --from=generate-build /api/inspect/*.pb.go /pkg/machinery/api/inspect/
-COPY --from=go-generate /pkg/machinery/config/types/v1alpha1/*_doc.go /pkg/machinery/config/types/v1alpha1/
```

Comment:
I hope this is not going to be a problem (copying all same files over itself), but I don't see why we need this change right now?

Reply:
This works and does not cause any problems.

That was a part of other work ("worker" vs "join"), but I think it belongs there because we call `go generate` on several packages, and we should copy them all. Otherwise, `//go:generate` results in some other files will be lost.